src/player.py

```diff
@@ -1,2 +1,51 @@
 # Write a class to hold player information, e.g. what room they are in
 # currently.
+from room import Room
+from item import Light, Food
+import os
+
+
+class Player():
```

Comment:
This code is very clean and readable, I really like how you implemented os.system('clear') in your travel method!